packages/kit/src/views/Discovery/pages/Browser/Browser.native.tsx

```
@@ -32,6 +40,8 @@
 import { useDebugComponentRemountLog } from '@onekeyhq/shared/src/utils/debug/debugUtils';
 import { EAccountSelectorSceneName } from '@onekeyhq/shared/types';
 
+import backgroundApiProxy from '../../../../background/instance/backgroundApiProxy';
```

ESLint

[error] Too many parent references (4 while maximum of 3 allowed)

(import-path/parent-depth)